cmd/operator-sdk/alpha/scorecard/cmd.go

```diff
@@ -52,6 +53,7 @@ func NewCmd() *cobra.Command {
 				Namespace:      namespace,
 				BundlePath:     bundle,
 				SkipCleanup:    skipCleanup,
+				Parallel:       parallel,
```

Comment:
I think we would want this to be something that is configurable via the config file. Without that, pipelines running scorecard against arbitrary bundles would have to do the most conservative thing, which would be to run everything serially.

That then begs the question, "where in the config file?" I think it needs to apply to a certain set of tests, so some tests could be in parallel and other tests could be in series. This would give users ultimate flexibility in organizing their scorecard runs, and pipelines could run tests in the most optimal way possible.

Reply:
my thought so far is to add a label onto each test in the current config file to indicate it could run in parallel, then when tests are selected, the selected tests that are labeled parallel could be executed together, any other tests would be executed serially afterwards. 
```
- name: "basic-check-spec"
  image: quay.io/operator-framework/scorecard-test:dev
  entrypoint:
  - scorecard-test
  - basic-check-spec
  labels:
    suite: basic
    parallel: true
    test: basic-check-spec-test
```

Reply:
What about the case where:
- test1 and test2 can run in parallel
- testA and testB can run in parallel
- test1 cannot run at the same time as testA

Or any other variation like that. I think it needs to be more explicit.

I keep going back to something like this:
```yaml
stages:
- tests:
  - name: "olm-bundle-validation"
    image: quay.io/operator-framework/scorecard-test:dev
    entrypoint:
    - scorecard-test
    - olm-bundle-validation
    labels:
      suite: olm
      test: olm-bundle-validation
  - name: "olm-crds-have-validation"
    image: quay.io/operator-framework/scorecard-test:dev
    entrypoint:
    - scorecard-test
    - olm-crds-have-validation
    labels:
      suite: olm
      test: olm-crds-have-validation
- tests:
  - name: "basic-check-spec"
    image: quay.io/operator-framework/scorecard-test:dev
    entrypoint:
    - scorecard-test
    - basic-check-spec
    labels:
      suite: basic
      test: basic-check-spec
- tests:
  - name: "olm-crds-have-resources"
    image: quay.io/operator-framework/scorecard-test:dev
    entrypoint:
    - scorecard-test
    - olm-crds-have-resources
    labels:
      suite: olm
      test: olm-crds-have-resources
- tests:
  - name: "olm-spec-descriptors"
    image: quay.io/operator-framework/scorecard-test:dev
    entrypoint:
    - scorecard-test
    - olm-spec-descriptors
    labels:
      suite: olm
      test: olm-spec-descriptors
  - name: "olm-status-descriptors"
    image: quay.io/operator-framework/scorecard-test:dev
    entrypoint:
    - scorecard-test
    - olm-status-descriptors
    labels:
      suite: olm
      test: olm-status-descriptors
```

In this case, we would:
1. Run `olm-bundle-validation` and `olm-crds-have-validation` in parallel and wait for them to complete.
2. Run `basic-check-spec` and wait for it to complete
3. Run `olm-crds-have-resources` and wait for it to complete
4. Run `olm-spec-descriptors` and `olm-status-descriptors` in parallel and wait for them to complete.

The idea is that every test in each stage can be run in parallel, but each stage is run in series. This type of layout would also let us add more fields at each layer to add more features if/when they come up.
```yaml
customGlobalConfig: customGlobalValue
stages:
- customStageConfig: customStageValue
  tests: []
```